server/src/main/java/com/objectcomputing/checkins/security/permissions/Permissions.java

```diff
@@ -15,6 +15,7 @@ public enum Permissions {
     CAN_VIEW_ANNIVERSARY_REPORT,
     CAN_VIEW_BIRTHDAY_REPORT,
     CAN_VIEW_PROFILE_REPORT,
+    CAN_VIEW_CHECKINS_REPORT,
```

Comment:
One more small thing needed with this. We need to load this permission into the DB and assign it to the ADMIN role in the load test data sql file.